bodhi/server/models/models.py

```diff
@@ -1708,6 +1712,18 @@ def last_modified(self):
         return possibilities[-1]  # Return the last one
 
     @property
+    def has_negative_karma(self):
+        """Check for negative karma, Returns True if the update has negative karma"""
+        feedback = defaultdict(int)
+        for comment in self.comments:
```

Comment:
It does appear that the comments are ordered by timestamp:

https://github.com/fedora-infra/bodhi/blob/3fe3e21946d43d895e38df34b6626b531946676c/bodhi/server/models/models.py#L658-L659